parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -147,8 +190,19 @@ public void writeTo(DataOutput out) throws IOException {
   }
 
   private static class FromStringBinary extends ByteArrayBackedBinary {
-    public FromStringBinary(byte[] value) {
-      super(value);
+    public FromStringBinary(String value) {
+      super(getBackingBytes(value), true);
+    }
+
+    private static byte[] getBackingBytes(String value) {
```

Comment:
Minor: This doesn't really return the backing bytes for the string, it returns the string encoded as UTF8 bytes. It would be better to name the method something like `encodeUtf8`. But it's private, so this isn't a big deal.